spec/services/code_climate/projects_without_c_c_spec.rb

```diff
@@ -35,9 +59,17 @@
                snapshot_time: Time.zone.now.ago(10.weeks)
       end
 
-      it 'returns project' do
-        projects = subject
-        expect(projects.first).to eq project
+      context 'and the project activity does not match' do
+        let(:project) { create :project, :with_activity, :internal, language: ruby_lang }
+        let(:period) { 2 }
```

Comment:
The problem I see here is that this test depends on a factory to set a specific attributes so that `the project activity does not match`. I'm ok with the trait `with_activity` but I'd like this place to be clear that the activity it's setting is older than the `period`